internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -1053,6 +1063,18 @@ func (r *ClusterDeploymentsReconciler) populateEventsURL(log logrus.FieldLogger,
 	return nil
 }
 
+func (r *ClusterDeploymentsReconciler) populateLogsURL(log logrus.FieldLogger, clusterInstall *hiveext.AgentClusterInstall, c *common.Cluster) error {
+	if *c.Status != models.ClusterStatusInstalled && !swag.IsZero(c.ControllerLogsCollectedAt) {
```

Comment:
No need for the check for `!swag.IsZero(c.ControllerLogsCollectedAt)` , it is done in `setControllerLogsDownloadURL`

Reply:
it currently isn't done there but I will add it to there